controlpanel/frontend/views/release.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.urls import reverse_lazy
+from django.views.generic.base import RedirectView
+from django.views.generic.edit import DeleteView, UpdateView, CreateView
+from django.views.generic.list import ListView
+from rules.contrib.views import PermissionRequiredMixin
+from django.http.response import HttpResponseRedirect
+
+from controlpanel.api.models import Tool
+from controlpanel.frontend.forms import ToolReleaseForm
+
+
+class ReleaseList(LoginRequiredMixin, PermissionRequiredMixin, ListView):
+    """
+    Used to display a list of releases of the tools available in the control
+    panel application.
+    """
+    context_object_name = 'releases'
+    model = Tool 
+    permission_required = 'api.list_release'
+    queryset = Tool.objects.all()
```

Comment:
Are `queryset` and `ordering` required when `model` is set? 
I suppose it would use ordering from `Tool` model if not right? (less is more but this is not an issue with the code, more of a question)

Reply:
The model is used for introspection so Django gets to know about the fields, their descriptions and constraints when doing things like, say, a `ModelForm` (based on such metadata).

The default Django ordering would apply, which is PK IIRC. It's easy to change. I suspect alphabetical order on the `name` field and then `release` in `Tools` would be a sensible default..? Happy to make this happen.

Reply:
Yeah I like the ordering you chose here, it makes a lot of sense so if you decide to move it to the model I'll be more than happy with that ☺️

Reply:
The ordering is already in place - see line 24. ;-)